### ISSUE TITLE
Erro when installing via Cocoapods

### ISSUE THREAD

**Issue by LeoValentim** [SwiftyJSON/SwiftyJSON#853]
I add this line in podfile "pod 'SwiftyJSON'" and run the command "pod install".

```
[!] Error installing SwiftyJSON
[!] /usr/bin/git clone https://github.com/SwiftyJSON/SwiftyJSON.git /var/folders/hn/kn07xxv11yjdpz6qgczwyb9m0000gn/T/d20170531-5321-1u0cb4j --template= --single-branch --depth 1 --branch v3.1.1

Cloning into '/var/folders/hn/kn07xxv11yjdpz6qgczwyb9m0000gn/T/d20170531-5321-1u0cb4j'...
warning: Could not find remote branch v3.1.1 to clone.
fatal: Remote branch v3.1.1 not found in upstream origin
```

**Comment by matthew-an** [comment]
The latest available version is 3.1.4. Please check your Podfile and try again. If possible, post your Podfile here.

**Comment by wongzigii** [comment]
Hey @LeoValentim This is related to this issue https://github.com/SwiftyJSON/SwiftyJSON/issues/839
